lib/shared/addon/components/notifier/modal-new-edit/template.hbs

```diff
@@ -0,0 +1,227 @@
+<div class="row nav nav-boxes mv-30 notifier-types">
+  {{#each types as |t|}}
+    <a
+      {{action "switchType" t.type}}
+      class="col span-2 nav-box-item driver {{if (eq t.type 'smtp') 'email' t.type}} {{if (eq t.type currentType) 'active' ''}}"
+      disabled={{t.disabled}}
+      alt={{t.type}}
+    >
+      <p>{{t.label}}</p>
+    </a>
+  {{/each}}
+</div>
+<hr />
+
+{{#if (eq currentType 'slack')}}
+  <div class="row">
+    <div class="col span-12 mt-0 mb-0">
+      {{form-name-description
+          name=model.displayName
```

Comment:
`displayName` sounds like something the API should be mapping to a field called `name`.

Reply:
The backend will update `displayName` to `name` and then I will change it to name.